### PR TITLE
feat: Allow Prisma client to be set on constructor

### PR DESCRIPTION
**Pull request by Zxilly**
Closes https://github.com/node-casbin/prisma-adapter/issues/24

### CODE REVIEW

**Comment by Zxilly** [comment]
@dimino @hsluoyz plz review

**Comment by Sefriol** [comment]
Actually did the review earlier, but forgot to submit it..

File: src/adapter.ts [hunk 1]
```diff
@@ -9,8 +9,14 @@ export class PrismaAdapter implements Adapter {
   #option?: Prisma.PrismaClientOptions;
   #prisma: PrismaClient;
 
-  constructor(option?: Prisma.PrismaClientOptions) {
+  constructor(
+    option?: Prisma.PrismaClientOptions,
```
[the remark this answers is not in the document]
Comment (hsluoyz) [reply src/adapter.ts:13]: > EDIT2: Alternative could also be to introduce a new interface which has optional parameters for options or the client

This way looks good, we should add another constructor method for it. @Zxilly plz make a new PR to fix this.

**Comment by dimino** [comment]
LGTM thanks a ton for jumping on this so quickly!

**Pull request merged**